soc/arm/ti_k3/Kconfig

```diff
@@ -0,0 +1,19 @@
+# Texas Instruments K3 Family
+#
+# Copyright (c) 2023 Texas Instruments Incorporated
+# Copyright (c) 2023 L Lakshmanan
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_FAMILY_TI_K3
+	bool
```

Comment:
please add some info in front of the bool

Reply:
I think these are intentionally hidden/promptless configs:
https://docs.zephyrproject.org/latest/build/kconfig/tips.html#assignments-to-promptless-symbols-in-configuration-files
Most `config SOC_FAMILY_*` / `config SOC_PART_NUMBER_*` configs follow this pattern